src/services/AbstractDFG.jl

```diff
@@ -472,14 +516,16 @@ Notes:
 - used by both factor graph variable and Bayes tree clique logic.
 """
 function isInitialized(var::DFGVariable; key::Symbol=:default)::Bool
-  solverData(var, key) != nothing && return solverData(var, key).initialized
-  return false
-end
-function isInitialized(fct::DFGFactor; key::Symbol=:default)::Bool
-  solverData(var, key) != nothing && return solverData(fct, key).initialized
-  return false
```

Comment:
DFG factor does not have this field if I remember correctly

Reply:
DFGFactor has thus far not required an `initialized` field, but does have an `eliminated` field which operates roughly in the same area in IIF.

Reply:
Should we open an issue for this? do you need something similar for eliminated, `isEliminated`. I thought it was a copy and pate error. since the field does not exists.

Reply:
I'm in two minds about an issue here.  Yes it was a copy paste, but maybe when we get to the performance improvements the concept of and "initialized" factor starts to exist, things like allocate the predicted memory for in-place operations, etc.  Do you have any strong feelings either way?